examples/cpp-simple/server.cc

```diff
@@ -16,101 +16,115 @@
 // This example really does nothing other than show how to integrate
 // the C++ SDK.
 
-#include <iostream>
-#include <thread>
 #include <agones/sdk.h>
 #include <grpc++/grpc++.h>
+#include <iostream>
+#include <thread>
 
 // send health check pings
 void doHealth(agones::SDK *sdk) {
```

Comment:
I realize this function name existed prior to your PR, but it's casing doesn't follow the [Google C++ Style Guide](https://google.github.io/styleguide/cppguide.html). This should be spelled `DoHealth`, and similarly it should be `WatchUpdates`.

If you want to fix these now, that's fine. If you'd rather not touch them, that's fine too. I'll let you decide.

Reply:
I've fixed them.